jerry-core/parser/js/js-parser-expr.c

```diff
@@ -633,6 +633,28 @@ parser_parse_class (parser_context_t *context_p, /**< context */
 } /* parser_parse_class */
 #endif /* !CONFIG_DISABLE_ES2015_CLASS */
 
+#ifndef CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER
+/**
+ * Parse object method.
```

Comment:
Please add more description, add the related section number as a note to the description.